extras/uwsgi/uwsgi.service

```diff
@@ -1,13 +0,0 @@
-[Unit]
-Description=uWSGI Emperor
-
-[Service]
-ExecStart=/usr/bin/uwsgi --emperor /etc/uwsgi/apps-enabled --logto /var/log/uwsgi/access.log
-RuntimeDirectory=uwsgi
-Restart=always
-KillSignal=SIGQUIT
-Type=notify
-NotifyAccess=all
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
Since we don't need to deal with uwsgi anymore, I picked the second option and bloated the mongodb roles instead.
https://github.com/ideascube/ansiblecube/tree/issue-25-uwsgi-cleanup